### PR TITLE
Feature/lep permission

### PR DESCRIPTION
**Pull request by akoidan**

### CODE REVIEW

File: src/main/java/com/icthh/xm/tmf/ms/communication/config/ApplicationProperties.java [hunk 1]
```diff
@@ -1,14 +1,33 @@
 package com.icthh.xm.tmf.ms.communication.config;
 
+//import com.icthh.xm.commons.lep.TenantScriptStorage;
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/communication/config/ApplicationProperties.java]: unnecessary import

File: src/main/java/com/icthh/xm/tmf/ms/communication/config/LepConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import com.icthh.xm.commons.lep.TenantScriptStorage;
+import com.icthh.xm.commons.lep.spring.EnableLepServices;
+import com.icthh.xm.commons.lep.spring.web.WebLepSpringConfiguration;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.ResourceLoader;
+
+@Configuration
+@EnableLepServices(basePackages = "your.base.package.with.services")
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/communication/config/LepConfiguration.java]: change to the real package

File: src/main/resources/config/application-dev.yml [hunk 3]
```diff
@@ -104,13 +106,13 @@ jhipster:
             queue-size: 512
 oauth2:
     signature-verification:
-        public-key-endpoint-uri: http://config/oauth/token_key
+        public-key-endpoint-uri: http://config/api/token_key
         #ttl for public keys to verify JWT tokens (in ms)
```
Comment (vkirichenko) [inline src/main/resources/config/application-dev.yml]: also need add this configuration to the app-prod.yml (fix wiki if needed)

File: src/main/java/com/icthh/xm/tmf/ms/communication/config/WebMvcConfiguration.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+import com.icthh.xm.commons.web.spring.config.XmWebMvcConfigurerAdapter;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.PathMatchConfigurer;
+
+import java.util.Collections;
+import java.util.List;
+
+@Configuration
+public class WebMvcConfiguration extends XmWebMvcConfigurerAdapter {
```
Comment (sergeysenja1992) [inline src/main/java/com/icthh/xm/tmf/ms/communication/config/WebMvcConfiguration.java]: we have the same configuration in commons why we copy paste it?

**Pull request merged**